package.json

```diff
@@ -40,61 +40,57 @@
   ],
   "dependencies": {
     "@launchpadlab/lp-hoc": "^5.0.4",
-    "classnames": "^2.2.5",
-    "cleave.js": "^1.5.3",
-    "filter-invalid-dom-props": "^2.0.0",
-    "lodash": "^4.17.4",
-    "moment": "^2.19.3",
-    "prop-types": "^15.5.8",
-    "react-color": "^2.14.0",
+    "classnames": "^2.3.1",
+    "cleave.js": "^1.6.0",
+    "filter-invalid-dom-props": "^2.1.0",
+    "lodash": "^4.17.21",
+    "moment": "^2.29.2",
+    "prop-types": "^15.8.1",
+    "react-color": "^2.19.3",
     "react-datepicker": "^1.0.3",
-    "react-modal": "^3.11.2",
+    "react-modal": "^3.14.4",
     "react-router": "^3.2.1",
-    "react-switch": "^5.0.1",
-    "redux-flash": "^2.0.1"
+    "react-switch": "^6.0.0",
+    "redux-flash": "^2.0.2"
   },
   "devDependencies": {
-    "@babel/cli": "^7.2.3",
-    "@babel/core": "^7.4.0",
-    "@babel/polyfill": "^7.4.3",
-    "@launchpadlab/babel-preset": "^2.1.0",
-    "@launchpadlab/eslint-config": "^2.4.1",
-    "@size-limit/preset-small-lib": "^3.0.0",
-    "@storybook/addon-a11y": "^5.2.5",
-    "@storybook/addon-actions": "^5.0.6",
-    "@storybook/addon-knobs": "^5.2.5",
-    "@storybook/addon-links": "^5.0.6",
-    "@storybook/addon-storysource": "^5.2.5",
-    "@storybook/addon-viewport": "^5.2.5",
-    "@storybook/addons": "^5.0.6",
-    "@storybook/react": "^5.0.6",
-    "babel-loader": "^8.0.0",
-    "bourbon": "^4.3.3",
-    "bourbon-neat": "^1.8.0",
-    "css-loader": "^3.0.0",
-    "documentation": "^12.1.1",
+    "@babel/cli": "^7.17.6",
+    "@babel/core": "^7.17.9",
+    "@launchpadlab/babel-preset": "^2.1.1",
+    "@launchpadlab/eslint-config": "^2.7.0",
+    "@size-limit/preset-small-lib": "^7.0.8",
+    "@storybook/addon-a11y": "^6.4.20",
+    "@storybook/addon-actions": "^6.4.20",
+    "@storybook/addon-knobs": "^6.4.0",
+    "@storybook/addon-links": "^6.4.20",
+    "@storybook/addon-storysource": "^6.4.20",
+    "@storybook/addon-viewport": "^6.4.20",
+    "@storybook/addons": "^6.4.20",
+    "@storybook/react": "^6.4.20",
+    "babel-loader": "^8.2.4",
+    "bourbon": "^7.2.0",
+    "bourbon-neat": "^4.0.0",
+    "core-js": "^3.21.1",
+    "css-loader": "5.2.7",
+    "documentation": "^13.2.5",
     "enzyme": "^3.2.0",
     "enzyme-adapter-react-16": "^1.1.0",
-    "eslint": "^6.2.1",
-    "husky": "^4.2.1",
+    "eslint": "^8.12.0",
+    "husky": "^7.0.4",
     "jest": "^25.1.0",
     "jest-canvas-mock": "^2.1.1",
-    "node-sass": "^4.11.0",
-    "react": "^16.8.0",
-    "react-dom": "^16.2.0",
+    "node-sass": "6.0.1",
+    "react": "16.14.0",
+    "react-dom": "16.14.0",
```

Comment:
Is there a specific reason why these went from flexible to pinned?

Reply:
😄 When one runs `yarn upgrade X@latest` the latest version is written to the `package.json`. All of these were upgraded with `yarn upgrade` so that the `yarn.lock` file remained correct.  I must admit I was confused on the "philosophy" when I started working on these templates. I originally changed all of these to minor versions to allow flexibility and then reverted to what was originally present, which seemed to be a mix of pinned and flexible versions. Let me know what you think. I could see updating these to at least a pinned minor version (middle number). So, for example, `react-dom` would remain as is but `documentation` would be updated to `^13.2.0` or even `^13.2`.

Reply:
Can you walk me through the rationale for pinning to a minor version?

Reply:
"Pinning to a minor version" may have been a misstatement on my part. I meant pinning the minor version element but leaving the patch version as 0. That would mean any version within that minor version would be allowed. Again, I'm not sure I understand the philosophy. I assume we're worried that major versions may introduce incompatibilities and so don't allow that to change. Minor versions should be less risky but, I feel, still should be explicitly controlled by us. The patch version should be able to be updated without issue. So that would mean "flexible" versions for the patch version only. Honestly, not sure how to achieve that using `yarn` without explicitly editing the `package.json` file after doing an upgrade to unpin the patch version. Maybe we need to discuss.

Reply:
I now see what you're referring to now. The carat (^) is missing from a few of these. I'll fix all of those but leave the version numbers as is for now, unless you feel otherwise.

Reply:
If you want to pin to the minor version, you can use `~` (refer to https://stackoverflow.com/a/22345808). However, I would prefer to stick with being more flexible by pinning the major version using the caret. If a library follows [semver](https://semver.org/) appropriately, I would think that we would welcome minor updates.

All that being said, it doesn't really matter for this library because it has a `yarn.lock` file! Where this really comes into play is the client-template, which deletes the `yarn.lock` and installs whatever is the latest in the acceptable range defined in `package.json`. Again, happy to discuss if there are concerns about being flexible with minor version updates

Reply:
Thanks. Yeah, agree that is less of an issue here but I didn't intend to pin to a version either. That happened when I upgraded certain libraries to their "non-latest" values. Glad you noticed that! Anyway, fixed.